### PR TITLE
Feature/version 2.0.2

### PR DESCRIPTION
**Pull request by q-litzler**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -28,4 +29,8 @@ allprojects {
         maven { url = URI("https://kotlin.bintray.com/kotlinx") }
         maven { url = URI("https://dl.bintray.com/kotlin/ktor") }
     }
+}
+
+tasks.withType<Test> {
+    maxParallelForks = Runtime.getRuntime().availableProcessors().minus(1).coerceAtLeast(1)
```
Comment (PLNech) [inline build.gradle.kts]: Why that? 🤔

File: gradle.properties [hunk 2]
```diff
@@ -1,4 +1,5 @@
 android.enableJetifier=true
 android.useAndroidX=true
 kotlin.code.style=official
-android.enableUnitTestBinaryResources=true
\ No newline at end of file
+android.enableUnitTestBinaryResources=true
+org.gradle.parallel=true
```
Comment (PLNech) [inline gradle.properties]: Sounds good but isn't it defeated by the line I first commented?

File: build.gradle.kts (hunk 1 above)
Comment (q-litzler) [reply build.gradle.kts]: This allows to run the test suite concurrently, using all processors (minus one) on your machine

File: gradle.properties (hunk 2 above)
Comment (q-litzler) [reply gradle.properties]: From what I understood: 

`org.gradle.parallel` allows parallelism for `build`

Comment (q-litzler) [reply gradle.properties]: https://guides.gradle.org/performance/#parallel_execution

File: build.gradle.kts (hunk 1 above)
Comment (PLNech) [reply build.gradle.kts]: Oh right, nice!

**Pull request merged**